servicebroker/test/routes/binding_test.js

```diff
@@ -112,14 +148,38 @@ describe('binding RESTful API', function() {
           .expect('Content-Type', /json/)
           .expect({})
           .end(function(err, res) {
+            expect(res.body.credentials.custom_metrics.username).to.equal('newusername');
+            expect(res.body.credentials.custom_metrics.password).to.equal('newpassword');
+
             binding.count({ where: { bindingId: bindingId } }).then(function(countRes) {
               expect(countRes).to.equal(1);
               done();
             })
           });
       });
+
+      it("failed to create a new binding because of credential generation failure", function(done) {
```

Comment:
The binding creation only fails when enableCustomMetrics is true.

Reply:
in test settings enableCustomMetrics is true only